presto-main-base/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -101,6 +104,21 @@ public class TaskManagerConfig
     private Duration highMemoryTaskKillerFrequentFullGCDurationThreshold = new Duration(1, SECONDS);
     private double highMemoryTaskKillerHeapMemoryThreshold = 0.9;
     private boolean enableEventLoop;
+    private Duration slowMethodThresholdOnEventLoop = new Duration(500, MILLISECONDS);
+
+    @Min(100_000_000L)
+    @Max(10_000_000_000L)
+    public long getSlowMethodThresholdOnEventLoop()
+    {
+        return slowMethodThresholdOnEventLoop.roundTo(NANOSECONDS);
+    }
+
+    @Config("task.event-loop-slow-method-threshold")
+    public TaskManagerConfig setSlowMethodThresholdOnEventLoop(Duration slowMethodThresholdOnEventLoop)
+    {
+        this.slowMethodThresholdOnEventLoop = slowMethodThresholdOnEventLoop;
+        return this;
+    }
```

Comment:
the roi on this config is not much. lets just add runtime metric

Reply:
discussed offline. this log will have the mehtod name details.. that will be helpful for debugging.